packages/sirius-web/backend/sirius-web/src/test/java/org/eclipse/sirius/web/application/controllers/portal/PortalControllerIntegrationTests.java

```diff
@@ -83,36 +105,157 @@ public class PortalControllerIntegrationTests extends AbstractIntegrationTests {
     private IGivenCommittedTransaction givenCommittedTransaction;
 
     @Autowired
-    private IRepresentationDataRepository representationDataRepository;
+    private IRepresentationContentRepository representationContentRepository;
 
     @Autowired
     private LayoutPortalMutationRunner layoutPortalMutationRunner;
 
     @Autowired
     private AddPortalViewMutationRunner addPortalViewMutationRunner;
 
+    @Autowired
+    private RemovePortalViewMutationRunner removePortalViewMutationRunner;
+
+    @Autowired
+    private IGraphQLRequestor graphQLRequestor;
+
     @BeforeEach
     public void beforeEach() {
         this.editingContextEventProcessorRegistry.getEditingContextEventProcessors().stream().map(IEditingContextEventProcessor::getEditingContextId)
                 .forEach(this.editingContextEventProcessorRegistry::disposeEditingContextEventProcessor);
     }
 
     @Test
-    @DisplayName("Given a portal, when we subscribe to portal events, then the current state of the portal is sent")
-    @Sql(scripts = { "/scripts/initialize.sql" }, executionPhase = Sql.ExecutionPhase.BEFORE_TEST_METHOD)
+    @DisplayName("Given a portal containing a view, when we delete the portal representation, then it is removed")
+    @Sql(scripts = { "/scripts/initialize.sql" }, executionPhase = Sql.ExecutionPhase.BEFORE_TEST_METHOD, config = @SqlConfig(transactionMode = SqlConfig.TransactionMode.ISOLATED))
     @Sql(scripts = { "/scripts/cleanup.sql" }, executionPhase = Sql.ExecutionPhase.AFTER_TEST_METHOD, config = @SqlConfig(transactionMode = SqlConfig.TransactionMode.ISOLATED))
-    public void givenPortalWhenWeSubscribeToPortalEventsThenCurrentStateOfThePortalIsSent() {
+    public void removePortalViewFromPortal() {
```

Comment:
Please respect the naming conventions of the tests, this comment is valid everywhere you have created a new test